Makefile

```diff
@@ -66,6 +66,7 @@ BUILD_FLAGS+=-tags $(BUILD_TAGS)
 endif
 
 TEST_FLAGS=-race -shuffle=on -timeout 10m
+JS_TEST_FLAGS=-exec="$$(go env GOROOT)/misc/wasm/go_js_wasm_exec" -shuffle=on -timeout 10m
```

Comment:
question: `-race` is not relevant?

Reply:
It doesn't work because we are using `go_js_wasm_exec` to run the tests.